framework/aster-frame/src/smp/mod.rs

```diff
@@ -0,0 +1,129 @@
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
Compared with moving `src/task/processor.rs`, I have a preference for renaming the current `PROCESSOR` to `TASKMANAGER`, as the current `PROCESSOR` only manages the tasks that are currently running, rather than the state of the processor. However, this is beyond the scope of this PR and requires further discussion. To avoid confusion, I believe it may be best to refrain from introducing `src/processor` and to keep the current format. I think your point is very valid, and we may need to refactor the `PROCESSOR` to better align with its semantic meaning.